src/main/java/backend/Logic.java

```diff
@@ -188,9 +188,9 @@ public String getDefaultRepo() {
     }
 
     public void removeUnusedModels(Set<String> reposInUse) {
-        models.toModels().stream().map(Model::getRepoId).
-                filter(repoId -> !reposInUse.contains(repoId)).
-                forEach(repoIdNotInUse -> models.removeRepoModelById(repoIdNotInUse));
+        models.toModels().stream().map(Model::getRepoId)
+                .filter(repoId -> !reposInUse.contains(repoId.toLowerCase()))
+                .forEach(repoIdNotInUse -> models.removeRepoModelById(repoIdNotInUse));
```

Comment:
Although we take care of `toLowerCase` inside, it could be helpful to add a method's comment recommending at `reposInUse` should be normalized as a precondition.